models/aeif_psc_exp.h

```diff
@@ -78,19 +78,19 @@ The membrane potential is given by the following differential equation:
 
 .. math::
 
- C dV/dt= -g_L(V-E_L)+g_L\cdot\Delta_T\cdot\exp((V-V_T)/\Delta_T)-g_e(t)(V-E_e) \\
-                                                     -g_i(t)(V-E_i)-w +I_e
+ C dV/dt= -g_L(V-E_L)+g_L\cdot\Delta_T\cdot\exp((V-V_T)/\Delta_T) - w(t) + I_{syn}(t) + I_e
 
 and
 
 .. math::
 
  \tau_w \cdot dw/dt= a(V-E_L) -W
```

Comment:
Should be lower case `w` and a space?

Reply:
Fixed!

Reply:
Not sure how to actually do this, the "Resolve conversation" button seems to be missing..